### PR TITLE
getting close

### PR DESCRIPTION
**Pull request by mhurtado2**
please check his out

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -56,4 +54,58 @@ footer {
...
+    text-align: center;
+    flex-basis: 25%;
+  }
+
+
+
+
+
+
+
```
Comment (thegoodcorg) [inline styles/main.css]: gonna need you to delete this big dawg

**Review by NelsonTaylorLumpkin — APPROVED** [review]
lewkin gewwwd

**Review by thegoodcorg — CHANGES_REQUESTED** [review]
lots of css to be deleted at the bottom

**Review by bigjlr2004 — APPROVED** [review]
Looking great. six thumbs up

**Review by thegoodcorg — APPROVED** [review]
this is cleaner

**Pull request merged**